apps/web/src/components/PoolCandidatesTable/PoolCandidatesTable.tsx

```diff
@@ -852,97 +370,220 @@ const PoolCandidatesTable = ({
+            poolCandidate: { user },
+            skillCount,
+          },
+        },
+      }) =>
+        skillMatchDialogAccessor(
+          allSkills?.filter((skill) => filteredSkillIds?.includes(skill.id)) ??
+            [],
+          user.experiences?.filter(notEmpty) ?? [],
+          skillCount,
+          `${user.firstName} ${user.lastName}`,
+        ),
+    }),
+    columnHelper.accessor(({ poolCandidate: { user } }) => user.email, {
+      id: "email",
+      header: intl.formatMessage(tableMessages.email),
+      sortingFn: normalizedText,
+      cell: ({
+        row: {
+          original: {
+            poolCandidate: { user },
           },
-          {
-            label: intl.formatMessage(adminMessages.notes),
-            value: "notes",
+        },
+      }) => user.email,
+    }),
+    columnHelper.accessor(
+      ({ poolCandidate: { user } }) =>
+        currentLocationAccessor(user.currentCity, user.currentProvince, intl),
+      {
+        id: "currentLocation",
+        header: intl.formatMessage(tableMessages.currentLocation),
+        cell: ({
+          row: {
+            original: {
+              poolCandidate: { user },
+            },
           },
-        ]}
-        onColumnHiddenChange={(event) => {
-          handleColumnHiddenChange(
-            allColumnIds,
-            hiddenColumnIds ?? [],
-            setHiddenColumnIds,
-            event,
-          );
-        }}
-        hiddenColumnIds={hiddenColumnIds ?? []}
-      />
-      <div data-h2-radius="base(s)">
-        <Pending
-          fetching={fetching || fetchingSkills}
-          error={error || skillsError}
-          inline
-        >
-          <BasicTable
-            labelledBy="pool-candidate-table-heading"
-            title={title}
-            data={filteredData}
-            columns={columns}
-            onSortingRuleChange={handleSortingRuleChange}
-            sortingRule={sortingRule}
-            hiddenColumnIds={hiddenColumnIds ?? []}
+        }) =>
+          currentLocationAccessor(user.currentCity, user.currentProvince, intl),
+      },
+    ),
+    columnHelper.accessor(
+      ({ poolCandidate: { submittedAt } }) => accessors.date(submittedAt, intl),
+      {
+        id: "dateReceived",
+        enableColumnFilter: false,
+        header: intl.formatMessage(tableMessages.dateReceived),
+        sortingFn: "datetime",
+        cell: ({
+          row: {
+            original: {
+              poolCandidate: { submittedAt },
+            },
+          },
+        }) => submittedAt,
+      },
+    ),
+  ] as ColumnDef<PoolCandidateWithSkillCount>[];
+
+  return (
+    <Table<PoolCandidateWithSkillCount>
+      caption={title}
+      data={filteredData}
+      columns={columns}
+      isLoading={fetching || fetchingSkills}
+      hiddenColumnIds={["candidacyStatus", "notes"]}
+      search={{
+        internal: false,
+        label: intl.formatMessage({
+          defaultMessage: "Search pool candidates",
```

Comment:
@yonikid15 not sure if you have already sent this string for translation, but the French would be: "Chercher les candidats du bassin".